packages/expo-enclave/example/App.tsx

```diff
@@ -22,6 +31,34 @@ export default function App() {
     setBiometricSecurityLevel(await ExpoEnclave.getBiometricSecurityLevel());
   })();
 
+  const testTx = async () => {
+    const accName = "testdaimo";
+    const derPublicKey = (await ExpoEnclave.fetchPublicKey(accName)) as string;
+    const signer: SigningCallback = async (message) => {
+      // hex version of '\x19Ethereum Signed Message:\n32'
+      const hexifiedPrefix =
+        "19457468657265756d205369676e6564204d6573736167653a0a3332";
```

Comment:
todo: this code block can/should probably just be wrapped into the middleware callback in the daimo-userop package for cleanliness